services/workflows-service/src/common/filters/AllExceptions.filter.ts

```diff
@@ -73,7 +74,7 @@ export class AllExceptionsFilter extends BaseExceptionFilter {
     this.logger.error(message, {
       name: error.name,
       status,
-      error: errorRes,
+      error: inspect(errorRes),
```

Comment:
Why?

Reply:
Currently, winston doesnt write other attributes of error so we dont have stacktraces 
https://stackoverflow.com/questions/47231677/how-to-log-full-stack-trace-with-winston-3

Will add a tech debt task

Reply:
https://linear.app/ballerine/issue/BAL-1786